### PR TITLE
Files cache multi arch

### PR DESCRIPTION
**Pull request by bradmwilliams**
The multi-arch folks reached out about the release-controller not supporting their respective installer downloads when accessing the Release Status webpage. This PR adds support for the files-cache to pull the installers from their architecture specific release-namespace.

### CODE REVIEW

File: cmd/release-controller/main.go [hunk 1]
```diff
@@ -200,7 +200,7 @@ func (o *options) Run() error {
 	execReleaseInfo := NewExecReleaseInfo(client, config, o.JobNamespace, fmt.Sprintf("%s", releaseNamespace), imageCache.Get)
 	releaseInfo := NewCachingReleaseInfo(execReleaseInfo, 64*1024*1024)
 
-	execReleaseFiles := NewExecReleaseFiles(client, config, o.JobNamespace, fmt.Sprintf("%s", releaseNamespace), imageCache.Get)
+	execReleaseFiles := NewExecReleaseFiles(client, config, o.JobNamespace, fmt.Sprintf("%s", releaseNamespace), fmt.Sprintf("%s", releaseNamespace), imageCache.Get)
```
Comment (wking) [inline cmd/release-controller/main.go]: I see it's a prexisting pattern, but what's this `Sprintf` stuff about?  Isn't `releaseNamespace` already a `string`?

```console
$ git describe --always --dirty
03d8c65c
$ git --no-pager grep ReleaseNamespaces
cmd/release-controller/main.go:	ReleaseNamespaces []string
cmd/release-controller/main.go:	flag.StringSliceVar(&opt.ReleaseNamespaces, "release-namespace", opt.ReleaseNamespaces, "The namespace where the source image streams are located and where releases will be published to.")
cmd/release-controller/main.go:	if len(o.ReleaseNamespaces) == 0 {
cmd/release-controller/main.go:	releaseNamespace := o.ReleaseNamespaces[0]
cmd/release-controller/main.go:	for _, ns := range o.ReleaseNamespaces {
```

It's also not clear to me what the difference between `name` and `releaseNamespace` is, if we can pass the same value to both of them.  Can `NewExecReleaseFiles` grow a godoc comment?

Comment (bradmwilliams) [reply cmd/release-controller/main.go]: I have cleaned up the preexisting pattern, added a couple godoc comments, and refactored the annotation/labels to be consistently named with the files-cache

File: cmd/release-controller/info.go [hunk 2]
```diff
@@ -483,7 +485,7 @@ class FileServer(SimpleHTTPServer.SimpleHTTPRequestHandler):
             try:
                 self._present_default_content(name)
 
-                subprocess.check_output(["oc", "adm", "release", "extract", "--tools", "--to", name, "--command-os", "*", "registry.svc.ci.openshift.org/ocp/release:%s" % name],
+                subprocess.check_output(["oc", "adm", "release", "extract", "--tools", "--to", name, "--command-os", "*", "registry.svc.ci.openshift.org/RELEASE_NAMESPACE/release:%s" % name],
```
Comment (smarterclayton) [inline cmd/release-controller/info.go]: Just make this an environment variable, and set that on the pod definition.  Don't use string replacement.

**Comment by smarterclayton** [comment]
Just the one comment.

**Comment by bradmwilliams** [comment]
@smarterclayton Thanks for the review! I have switched the logic to use environment variables.

**Comment by smarterclayton** [comment]
Squash the env var commit into the appropriate commit, then this is fine.

**Comment by bradmwilliams** [comment]
> Squash the env var commit into the appropriate commit, then this is fine.

@smarterclayton I've squashed the 2 relevant commits.

**Pull request merged**